featureFlag/index.html

```diff
@@ -108,5 +108,16 @@
         </div>
       </form>
     </div>
+    <footer>
+      <p class="info-repo">
+        The contents of this website are deployed from this
+        <a
+          href="https://github.com/Real-Dev-Squad/website-dashboard"
+          target="_blank"
+          rel="noopener noreferrer"
+          >open sourced repo</a
+        >
+      </p>
+    </footer>
```

Comment:
Let us extract the footer code in one place and use it from there. Let's create a separate issue and track this

Reply:
sure we can. It has vanilla code so there is duplicate code everywhere.